external-idp/src/main/kotlin/com/pingidentity/idp/davinci/IdpCollector.kt

```diff
@@ -99,32 +101,33 @@ class IdpCollector : Collector<Nothing>, ContinueNodeAware, DaVinciAware, Reques
     /**
      * Authorizes the user using the specified IdP.
      *
+     * @param redirectUri The redirect URI for the browser-based IdP authorization.
```

Comment:
Same here...

Reply:
Update all from IdP authorization to IdP authentication.